langchain/document_loaders/recursive_url_loader.py

```diff
@@ -0,0 +1,86 @@
+from typing import Iterator, List, Optional, Set
+from urllib.parse import urlparse
+
+import requests
+from bs4 import BeautifulSoup
+
+from langchain.docstore.document import Document
+from langchain.document_loaders.base import BaseLoader
+
+
+class RecusiveUrlLoader(BaseLoader):
+    """Loader that loads all child links from a given url."""
+
+    def __init__(self, url: str, exclude_dirs: Optional[str] = None) -> None:
```

Comment:
Maybe exclude paths or patterns for the variable mame? Dirs remind me of the file system

Reply:
Ya, was thinking about this too. Will look into it!